libethereum/ChainParams.cpp

```diff
@@ -37,6 +59,16 @@ ChainParams::ChainParams(string const& _json, h256 const& _stateRoot)
     *this = loadConfig(_json, _stateRoot);
 }
 
+ChainParams::ChainParams(std::string const& _configJson, AdditionalEIPs const& _additionalEIPs)
+  : ChainParams(_configJson)
+{
+    lastForkAdditionalEIPs = _additionalEIPs;
```

Comment:
(Nit) Can we initialize this as a part of the initializers list?

Reply:
No, because these are members of the base class